docs/coordinates/solarsystem.rst

```diff
@@ -0,0 +1,45 @@
+.. include:: references.txt
+
+.. _astropy-coordinates-solarsystem:
+
+Solar System Ephemerides
+----------------------------
+
+`astropy.coordinates` can calculate the |SkyCoord| of some of the major
+solar system objects. This functionality requires the 
+`jplephem <https://pypi.python.org/pypi/jplephem>` package
```

Comment:
You need a ``_`` after the last backtick. You can check that the docs build locally with:

```
python setup.py build_sphinx
```

And they should appear in ``docs/_build/html/index.html``.